### DISCUSSION TITLE
Maiana as an autonomous wireless AIS solution

### DISCUSSION THREAD

**Discussion by nxmyoz** [peterantypas/maiana#42] [Ideas]
Hello dear Peter,
hello dear Maiana Community,

I follow your work on the board with great interest since it was available on OSH Park. Thank you for your efforts. It is amazing to see, how the development accelerated over the last months.

As someone who doesn't own a boat on which to put Maiana on, but frequently charters one, now that the form factor became so tiny, I wanted to share my ideas on a wireless and autonomous Maiana enhancement:

As you might know, not all charter boats (mainly Europe here) have all the nice navigational gadgets onboard. Newer or fresh yachts may have all the nice gadgets but are a horror from a sailing-handling point of view (like a caravan for water, that is, most of the time it is being handled by inexperienced skippers who most of the time anchor around and if they 'sail', they do so with the engine onboard instead the sails; so are new yachts for the charter segment: not mainly designed for sailing) being pricier at the same time. Or one finds a nice but aged yacht but with devices mostly from the early 2000s. Older yachts are nice.

The problem with older charter yachts are:

- you do not know who used it before,
- you do not know how it was used,
- although such a yacht complies with regulation (safety & navigation & equipment), that is everything seems to be onboard, yet sometimes not everything does work as it supposed to,
- sometimes charts are outdated or a bare minimum is provided (paper and electronic),
- sometimes, if a plotter is onboard, it does work but maybe from time to time.

All that led to bringing of own navigational gear: A laptop with OpenCPN with the latest charts for the area I'm sailing in (mostly Mediterranean Sea, Adriatic Sea, Baltic Sea or the North Sea), a "black-box"-type of NMEA gateway I can build on top of a SBC (at this point I'm using an Odroid-C2, because better open source support) with my own minimal "navigational"-Linux-distribution, which runs kplex, gpsd, at the moment AIS signal processing based on rtl-sdr and a pressure and temperature sensor. It is configured to be a "fire-and-forget" solution. Combined with a small-factor wireless router in combination with a 4G modem, all connected devices get the NMEA data. That way, for example I even get AIS in the Navionics app, which I use in the cockpit, as it runs on a ruggedized tablet (I would also like to use the OpenCPN app, yet it isn't ready for this kind of use). Together with a waterproof portable Garmin Gpsmap, I get a nice setup that meets my needs and which is very portable and reliable.

Using rtl-sdr is easy and cheap in comparison. Yet rtl-sdr seems not to be the right thing, at least at sea. I haven't figured out, how to increase the range of the reception of AIS signals, as it's not as easy to bring a wired rtl-sdr usb stick outside. I get maybe 3-5 nm. All the wiring would be a nightmare. Also, it seems Realtek's chipset consumes so much power, you do not need a heater then if it gets cold outside and from time to time because of this heat, it stops working. I get AIS signals as a proof of concept but if it works, it does so most unreliably.

So watching the progress of Maiana carefully, and looking at the power consumption and the size, I think it would be possible to create an attachable pipe-type Maiana device, that is powered by lithium-ion batteries, which can be charged by a solar panel, that can be wrapped around the housing of Maiana. Also the NMEA data could be sent via Bluetooth or Wi-Fi. 

I would also use a pipe for the housing of Maiana, the li-ion batteries and charger and the wireless module. The pipe can be longer but have a rather small diameter. One could wrap a flexible solar panel around the pipe which should theoretically provide enough to recharge the batteries. A not permanent attachment on a shroud or another suitable spot should be easy.

Most of the time, a boat is chartered for up to two weeks. It shouldn't be a problem to calculate the consumption of this setup in order to determine the dimensions and amount of batteries.

Since most of the time this device will be in European waters it will receive the AIS signals **but not transmit**. Charter yachts have all the different flags and owners, and bringing your own AIS transmitter onboard a charter yacht would be a regulatory nightmare, because most of the time you need to register a transmitter and if you do so, it must then comply with regulation and have all the necessary certification, as you are all well aware of (discussion #32), not to mention the case if there is an inspection... So, someone who is not the boat owner would have big problems registering such a device, even if it were certified... 

I plan to share all the results and material on this project.

What do you think about this? Do you have any suggestions?

Kind Regards ;)

**Comment by peterantypas** [comment]
I think the ideal enclosure for what you're describing is something similar to a radar reflector, suspended using a flag halyard up near the spreader. I did test MAIANA in this configuration and it works very well. The main challenge will be the mechanical design.

Given that you don't need TX functionality, you're really only looking for a 2 channel AIS receiver with low power consumption. Would you still need the GPS/GNSS? That draws more power than the rest of the system.

**Reply by nxmyoz** [reply]
Maiana comes with the GNSS module, so why not use it as a backup if needed, or even replace what I'm using now (less wiring and wires around the navigation table). And even with the GNSS module working, 18650 lithium batteries should provide enough power. In my consideration I think I came to about 4-6 18650 batteries I would need. There are enough boards for controlling charging of 18650 batteries out there.

I have an u-blox GNSS module attached to my SBC. It works great, I like how one can configure the chipset with u-blox tools or ubxtool shipped with gpsd.

However, in case of a situation I might need to transmit my own AIS signal, the board should have the GNSS module working. In this case even if the transmitter is not certified and registered as it should be, using it during emergency situations or bad weather (heavy fog?) won't/shouldn't lead to any problems. So the possibility of transmitting AIS is a nice to have option. Also, I would be free to set the MMSI, which in this case is important- another yacht, another MMSI...

The board at this moment is very suitable for this. I see no problems with the power. A right amount of 18650 li-batteries should be sufficient, even if I calculate for an uninterrupted (receive only) use for up to 14 days. And should I ever need to transmit, there would be enough power to reach the next harbour.

**Reply by peterantypas** [reply]
Ahhhh, so you might still need to transmit in exceptional situations ...

In that case, you will need to supply at least 9V to V_IN in order to get 8V on the PA. The other requirement is that your BLE/WiFi module controls the TX_OFF signal somehow (assuming you have provisioned MMSI, name, etc). 

Sounds like a fun project, would love to see how it progresses.

**Reply by nxmyoz** [reply]
The first step would be receiving only, that should be fairly easy to do (also because of the power supply in that case).

There is some time for contemplation about this, as the Silicon Labs chips are still not available...

**Reply by peterantypas** [reply]
I ran a preliminary power budget for this and it looks tough. MAIANA draws 70mA from the 3.3V rail during RX. Assuming a 3200mAh battery, that gives you about 48 hours, and that doesn't account for any RF module like ESP32. For this to work for 14 days, you will either need > 8 batteries or a solar panel that can charge an 18650 sufficiently during the day while operating the circuit at the same time.

Some optimizations might be possible in firmware. The design could  change to allow the GNSS to go to sleep periodically, and the RX duty cycle could decline from 100% to something like 10% when little movement is detected (you're at anchor). But I think you might be better off to dismount it and plug it into a charger at night. That's the approach that I would take.

**Comment by jrcardwell** [comment]
Have to ask why, when at anchor with no power, we need to transmit AIS at all?? Is this overkill?

Cheers all,

Jim

**Reply by revolteur** [reply]
On Power Consumption and optimization: Having the ability to configure the RX/Tx duty cycle for the AIS as well as the broadcast rate of NEMA data over Wi-Fi (ESP32) via a configuration page will come in handy to lower power consumption at anchor (ex: setting a Geofence using the GNSS) and while under sail. This is the kind of fun stuff that can be added along the way to make the system smarter. 

On Power:  Don't you think it's reasonable to assume that if a sailor is going to be at sea for an extended period of time (more than a daysail let's say) they would have some type of power and charging capabilities on the boat like batter/solar/wind/charger. Cutting the power cord would be cool but is it really needed when considering the entire system and its permanent install use case. The AIS is just one part of the system. 

Personally, I would have a dedicated power system (battery/solar panel/Charger) for the primary or backup AIS/NAV/COM. 
I would keep it fully isolated from the main power of the boat and not connected to anything on the masthead to help reduce catastrophic failure from a lightning strike.

**Reply by peterantypas** [reply]
I think a better market for a battery powered unit may be recreational fishing.

**Reply by kagouraki** [reply]
Hello guys. 
I am following this project for a while now and i have a v6 board and parts but no time yet to assemble. 
I wanted to say something about the whole wireless idea:

Every boat has, at least, a anchor light on the top of the mast. So we have 12v there. 
So no need for battery and solar etc. 
We can have the whole maiana on top of the mast and power it through the anchor light cable. 
In the breakoutboard we can have a mosfet to turn, throught the esp32, the anchor light on and off. Everyone has led bulbs now so no big consumption there. Then there will be controlled everyhing from the esp32. Program maiana ais details, tx off, nmea out. The on-off will be from the chart table. 
What do you think ?
